components/automate-cli/cmd/chef-automate/debug.go

```diff
@@ -91,8 +91,8 @@ func newDebugCmd() *cobra.Command {
 	logLevelCmd.Flags().StringVar(&logLevelCmdFlags.endpoint, "endpoint", "", "The endpoint the service is listening on")
 	logLevelCmd.Flags().DurationVar(&logLevelCmdFlags.connectionTimeout, "connection-timeout", 0, "Most time to wait to connect to service")
 
-	versionCmd.Flags().StringVar(&versionCmdFlags.endpoint, "endpoint", "", "The endpoint the service is listening on")
-	versionCmd.Flags().DurationVar(&versionCmdFlags.connectionTimeout, "connection-timeout", 0, "Most time to wait to connect to service")
+	getVersionCmd.Flags().StringVar(&versionCmdFlags.endpoint, "endpoint", "", "The endpoint the service is listening on")
```

Comment:
The flags were being set to the wrong versionCmd object, as a result of which, endpoint and connection timeout flags were coming up in docs under chef-automate version command. Corrected this